docs/source/concepts/building_bentos.rst

```diff
@@ -129,40 +162,92 @@ directory. Users can also opt to place a `.bentoignore` file in the directory wh
 run to achieve the same file exclusion during build. If not explicitly specified, nothing is excluded 
 from the build. Exclude is applied after include.
 
-Docker Options
-^^^^^^^^^^^^^^
-
-Options for generating the Docker image of the Bento service, such as selecting the base image and 
-enabling the use of GPU.
-
-.. todo::
-
-    Add Docker options examples
-
-
 Environment
 ^^^^^^^^^^^
 
-The `env` keyword argument specifies the Python version and dependencies required to deploy the bento. 
-If not explicitly specified, the build to automatically infer the PyPI packages required by the service 
-by recursively walking through all the dependencies. While the auto-infer in convenient, we still 
-recommend to define the required package and versions explicitly, to ensure more deterministic build 
+The `env` keyword argument specifies the Python version and dependencies required to deploy the bento.
+If not explicitly specified, the build to automatically infer the PyPI packages required by the service
+by recursively walking through all the dependencies. While the auto-infer in convenient, we still
+recommend to define the required package and versions explicitly, to ensure more deterministic build
 and deployment.
 
-.. todo::
-
-    Add Environment options examples
```

Comment:
@aarnphm Where is an example of some of these options? Isn't this just a combination of Python/Docker options now?